canary/Models/Message.cs

```diff
@@ -54,20 +54,18 @@ public Message(Record record, String type)
             switch (type)
```

Comment:
Should the hardcoded URLs in the `case` statements be replaced with references to the constants defined inside VRDR?

Reply:
Good thought, not something that should hold up this preview release